### PR TITLE
[webui] format code by eslint and prettier

### PR DESCRIPTION
**Pull request by Lijiaoa**
- [x] [stylelint](https://stylelint.io/user-guide/rules/about) reference
    - `yarn stylelint --fix` could almost format css/scss code

- [x] [prettier](https://prettier.io/docs/en/index.html)
- [x] add some eslint rule
        
```
 // no use `arrow-parens: [2, "as-needed", { "requireForBlockBody": true }]`,
 // because `"requireForBlockBody": true`  is conflict with prettier `arrowParens: 'avoid',`
 "arrow-parens": [2, "as-needed"],
 "no-empty": 2, // block is null will error
 "no-multiple-empty-lines": [2, { "max": 1 }], // max empty line is 1
```

### CODE REVIEW

File: src/webui/src/components/Modals/ChangeColumnComponent.tsx [hunk 1]
```diff
@@ -41,7 +47,7 @@ class ChangeColumnComponent extends React.Component<ChangeColumnProps, ChangeCol
         } else {
             if (source.includes(label)) {
                 // remove from source
-                const result = source.filter((item) => item !== label);
+                const result = source.filter(item => item !== label);
```
Comment (liuzhe-lz) [inline src/webui/src/components/Modals/ChangeColumnComponent.tsx:50]: I think this is less clear to human. It's hard to figure out associative order at first glance to me.

Comment (Lijiaoa) [reply src/webui/src/components/Modals/ChangeColumnComponent.tsx:50]: alwaysParens: `avoid | always`;
default val is avoid, not add () for arrow function.
So we change it to `always`?

File: src/webui/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (liuzhe-lz) [inline src/webui/.editorconfig:1]: Is this file used by a specific text editor? If yes it should not be committed to code tree.

File: src/webui/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```
Comment (liuzhe-lz) [inline src/webui/.editorconfig:8]: And I wonder why your files do not have trailing new line even if this is set to `true`.

File: src/webui/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+insert_final_newline = true
```
Comment (liuzhe-lz) [inline src/webui/.editorconfig:9]: And I wonder why your files do not have trailing new line even this is set to `true`...

File: src/webui/src/App.tsx [hunk 5]
```diff
@@ -117,39 +123,43 @@ class App extends React.Component<{}, AppState> {
...
+                <Stack className='contentBox'>
+                    <Stack className='content'>
                         {/* if api has error field, show error message */}
-                        {
-                            errorList.map((item, key) => {
-                                return (
-                                    item.errorWhere && <div key={key} className="warning">
-                                        <MessageInfo info={item.errorMessage} typeInfo="error" />
+                        {errorList.map((item, key) => {
+                            return (
```
Comment (liuzhe-lz) [inline src/webui/src/App.tsx:150]: Is `return` necessary here?

**Comment by liuzhe-lz** [comment]
I think we should discuss the rules in a meeting, just like what we have done for pylint rules.
For pylint, we listed all default rules violated by our code, and discussed them one by one.

File: src/webui/.editorconfig (hunk 4 above)
Comment (Lijiaoa) [reply src/webui/.editorconfig:9]: but `.tsx .scss` these files all have new line in last line.

File: src/webui/src/App.tsx (hunk 5 above)
Comment (Lijiaoa) [reply src/webui/src/App.tsx:150]: I also think so. We can use arrow function rules to standardize it.

**Review by ultmaster — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**